internal/pkg/aws/stepfunctions/stepfunctions.go

```diff
@@ -39,3 +40,15 @@ func (s *StepFunctions) StateMachineDefinition(stateMachineARN string) (string,
 
 	return aws.StringValue(out.Definition), nil
 }
+
+//Execute starts a state machine execution
+func (s *StepFunctions) Execute(stateMachineARN string) error {
+	_, err := s.client.StartExecution(&sfn.StartExecutionInput{
+		StateMachineArn: aws.String(stateMachineARN),
+	})
+	if err != nil {
+		return fmt.Errorf("execute state machine: %w", err)
```

Comment:
Do you think this may make the error message more helpful:
```suggestion
		return fmt.Errorf("execute state machine %s: %w", stateMachineARN, err)
```